Keywords.txt

```diff
@@ -1,5 +1,7 @@
+Vulnerability | Increases the amount of damage a target takes by their Vulnerability | 150 | 30 | 30
+Exposure | Multiplies the amount of damage a target takes by their exposure | 175 | 35 | 35
```

Comment:
Lets make the capitilzation consistnet here, exposure in the description should be capitialized